internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -104,7 +104,7 @@ func SchemaAdvancedConfigDS() *schema.Schema {
 					Computed: true,
 				},
 				"oplog_min_retention_hours": {
-					Type:     schema.TypeInt,
+					Type:     schema.TypeFloat,
```

Comment:
have you ckecked if there are plan changes for existing clusters?

Reply:
Yes, I did a manual test. I also updated the migration test to include this.